packages/valory/skills/abstract_round_abci/test_tools/integration.py

```diff
@@ -366,7 +368,7 @@ def setup(cls, **kwargs: Any) -> None:
         super().setup()
 
         # register gnosis contract
```

Comment:
Technically this pollutes the concerns. But we can clean this up in a next step.